### ISSUE TITLE
Sign up with email and passkey label throws HTTP 500 server error

### ISSUE THREAD

**Issue by heliocola** [ruby-passkeys/devise-passkeys-template#5]
### Steps to replicate the issue:
1. Clone `devise-passkeys-template`
2. Install ruby, bundler, run `bundle install`, db:migrate etc
3. Setup puma-dev
4. Run `bin/start-puma`

### Environment:
The scenario below is the same using devise-passkeys gem v0.1.0 from Rubygems as well as with the latest main branch on GitHub.
Gemfile:
> gem "devise-passkeys", github: "ruby-passkeys/devise-passkeys"
> gem "devise-passkeys"

### Expected behavior:
1. I am able to test the creation of an user, its passkey, and login with it.

### Current behavior:
1. Having a HTTP 500 server error trying to create an user:
> INFO: devise-passkeys (f2739a7a8470)
```
NameError (undefined local variable or method `relying_party' for #<Users::RegistrationsController:0x0000000000c350>):
  
devise-passkeys (f2739a7a8470) lib/devise/passkeys/controllers/registrations_controller_concern.rb:35:in `new_challenge'
actionpack (7.0.4.2) lib/action_controller/metal/basic_implicit_render.rb:6:in `send_action'
actionpack (7.0.4.2) lib/abstract_controller/base.rb:215:in `process_action'
actionpack (7.0.4.2) lib/action_controller/metal/rendering.rb:53:in `process_action'
actionpack (7.0.4.2) lib/abstract_controller/callbacks.rb:234:in `block in process_action'
activesupport (7.0.4.2) lib/active_support/callbacks.rb:118:in `block in run_callbacks'
```

Or with Rubygems devise-passkeys:
> devise-passkeys (0.1.0)
```
NameError (undefined local variable or method `relying_party' for #<Users::RegistrationsController:0x0000000000bea0>):
  
devise-passkeys (0.1.0) lib/devise/passkeys/controllers/registrations_controller_concern.rb:35:in `new_challenge'
```

**Comment by tcannonfodder** [comment]
Ack; good find! Fixing this now! The concern wasn't being included in the `RegistrationsController`

**Comment by tcannonfodder** [comment]
Fixed in #6
